src/pybpmn/parser.py

```diff
@@ -84,6 +84,10 @@ def parse_bpmn_img(self, bpmn_path: Path, img_path: Path) -> AnnotatedImage:
 
         if self.scale_to_ann_width:
             self.scale_anns_to_img_width_(anns, bpmn_path, img)
+        else:
```

Comment:
ich würde das else entfernen und das resize_arrows_to_min_wh aus der Methode scale_anns_to_img_width_ entfernen, und es dann für beide Varianten hier aufrufen.
Das Arrow Resizing gehört nicht in die Methode scale_anns_to_img_width_ weil diese eine andere Aufgabe hat, das hatte ich damals ausversehen dort integriert.